UIMenu.cs

```diff
@@ -1244,8 +1245,8 @@ public bool Visible
                 UpdateScaleform();
                 if (ParentMenu != null || !value) return;
                 if (!ResetCursorOnOpen) return;
-                Cursor.Position = new Point(Screen.PrimaryScreen.Bounds.Width/2, Screen.PrimaryScreen.Bounds.Height/2);
-                Function.Call(Hash._0x8DB8CFFD58B62552, 1);
+                Cursor.Position = new Point(System.Windows.Forms.Screen.PrimaryScreen.Bounds.Width/2, System.Windows.Forms.Screen.PrimaryScreen.Bounds.Height/2);
```

Comment:
I don't know why this is using windows forms, doesn't Screen.Resolution work like the others?